examples/complex_module/README.md

```diff
@@ -54,7 +54,7 @@ An example configuration for an Arm component, a Gizmo component, and a Summatio
       "name": "gizmo1",
       "type": "gizmo",
       "namespace": "acme",
-      "model": "viam:gizmo:mygizmo",
+      "model": "acme:demo:mygizmo",
```

Comment:
I changed this back to acme because I did not see a gizmo as an example of a Viam namespace supported component as per the RDK repo